src/PercentileStats.h

```diff
@@ -124,16 +125,55 @@ namespace QDirStat
         QRealList percentileList();
 
         /**
-         * Return a list (0..100) of all accumulated sizes between one
-         * percentile and the previous one.
+         * Returns both forms of percentile sums.
          **/
-        QRealList percentileSums();
+        PercentileSums percentileSums();
 
 
     protected:
 
 	QRealList _data;
-	bool	  _sorted;
+	bool	  _sorted;   
+    };
+
+    /**
+     * Container for individual and cumulative percentile sums.
+     **/
+    class PercentileSums
+    {
+    public:
+        /**
+         * Constructor.
+         **/
+        PercentileSums() {}
+
+        /**
+         * Returns the size of the sums.
+         **/
+        int size() const { return _ind.size(); }
+
+        /**
+         * Returns true if the sums are empty; otherwise, returns false.
+         **/
+        bool isEmpty() const { return _ind.isEmpty(); }
+
+        /**
+         * Return a lists (0..100) of all accumulated sizes between one percentile
+         * and the previous one
+         **/
+        const QRealList& individual() const { return _ind; }
```

Comment:
Please add a space before the `&` for references.

I know there are different styles to do that; with or without space before and/or after the `&` (or the `*` for pointers). But the important part is that it's consistent within the project.

Reply:
Sorry about that, checking myself deviating from other peoples code styles is definitely a weak point of mine.

Done.